src/Controller/EnMarche/Jecoute/News/AbstractNewsController.php

```diff
@@ -61,8 +62,13 @@ public function jecouteNewsCreateAction(
             $news->setZone($zones[0]);
         }
 
+        $options = ['zones' => $zones];
+        if (JecouteSpaceEnum::REFERENT_SPACE === $this->getSpaceName()) {
+            $options['is_notifiable'] = false;
+        }
```

Comment:
what about adding a new abstract method: `isNotifiableNews`?

```suggestion
        $options = [
          'zones' => $zones,
          'is_notifiable' => $this->isNotifiableNews(),
        ];
```